tests/common.py

```diff
@@ -41,6 +42,13 @@ def get_110_ccc_ff():
     return forcefield
 
 
+def fixed_overflowed(a):
+    """Refer to timemachine/cpp/src/kernels/k_fixed_point.cuh::FLOAT_TO_FIXED_ENERGY for documentation on how we handle energies and overflows"""
+    converted_a = np.int64(np.uint64(a))
+    assert converted_a != np.iinfo(np.int64).min, "Unexpected value for fixed point energy"
```

Comment:
Worth expanding the message or adding a comment here? (i.e. why can negative overflow not happen?)

Reply:
I think the documentation for `FLOAT_TO_FIXED_ENERGY` should cover this. This is just an indicator that the expectations are off.